test/concurrent.test.js

```diff
@@ -171,8 +176,8 @@ const runOneTest = async (ti, tj) => {
   // Verify initial states match.
   expect(ei.slateDoc.document.nodes.toArray().map(nodeToJSON))
     .toEqual(ej.slateDoc.document.nodes.toArray().map(nodeToJSON));
-  expect(toSlateDoc(ei.syncDoc).map(nodeToJSON))
-    .toEqual(toSlateDoc(ej.syncDoc).map(nodeToJSON));
+  expect(toSlateDoc(ei.syncDoc.get('document')).map(nodeToJSON))
```

Comment:
Both here and below: do you also want to compare "data"?

(Which brings up the side node: should there also be a test transform that yields a set_value operation, so the test is verifying behavior for concurrent/out-of-order application of such operations wrt all the other types of operations?)

Reply:
The comparison was added and new Transforms were added to test the behavior and propagation for the set_value function.